config/exchanges.json

```diff
@@ -576,7 +576,7 @@
     "parsers": {
       "exchange": "EIA.fetch_exchange"
     },
-    "rotation": 0
+    "rotation": 180
```

Comment:
I'm unsure if `CA-MB->US-MIDW-MISO` actually works? It's not showing anything right now. However, the rotation seems to be correct by analogy with `CA-BC->US-NW-BPAT` :+1:

Reply:
I think the exchanges only show if they have 'live' data, which is kind of a shame. EIA has data for this exchange, it just comes in later (8-24 hours delay), so it won't show on the map, but it will be included in historic calculations.